cmd/odfcli/set/recovery-profile_test.go

```diff
@@ -0,0 +1,25 @@
+package set
+
+import "testing"
+
+func Test_isValidArg(t *testing.T) {
+	type args struct {
+		args string
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{"Valid Input with 'high_client_ops' as arg", args{args: "high_client_ops"}, true},
+		{"Valid Input with 'balanced' as arg", args{args: "balanced"}, true},
+		{"Invalid input for mclock profile", args{args: "balance"}, false},
```

Comment:
Should probably test that all valid inputs are accepted.
Also, empty string test for corner case.